Gemfile.lock

```diff
@@ -34,8 +34,8 @@ GEM
     diff-lcs (1.2.5)
     em-socksify (0.2.0)
       eventmachine (>= 1.0.0.beta.4)
-    eventmachine (1.0.3)
-    eventmachine (1.0.3-java)
+    eventmachine (1.0.9.1)
+    eventmachine (1.0.9.1-java)
```

Comment:
I could not build eventmachine on my linux box without bumping the patch version.